### PR TITLE
feat: add default actions for Pods empty state

### LINKED ISSUE #8570: Default actions for Pods empty state
There should be two options in the following screen to allow the user to quickly get started:

Screenshot 2024-08-25 at 13 05 15

The first one should be a button:
"Start your first pod"

Ideally, we would also like to provide sample pods (we might be able to pull those from https://github.com/redhat-developer/podman-desktop-demo

Then in the screen, we display a label "OR" and the line bellow is the actual commande line we already have in the screen

### PR DESCRIPTION
**Pull request by dgolovin**
### What does this PR do?

Adds button 'Start my first pod' to Pods page empty screen

### Screenshot / video of UI

https://github.com/user-attachments/assets/88cdfa1d-4894-45dd-8e4f-cfe7f20e5c0a

### What issues does this PR fix or reference?

Fix #8570.

### How to test this PR?

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/renderer/src/lib/pod/PodEmptyScreen.spec.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/**********************************************************************
+ * Copyright (C) 2023-2024 Red Hat, Inc.
```
Comment (feloy) [inline packages/renderer/src/lib/pod/PodEmptyScreen.spec.ts]:
```suggestion
 * Copyright (C)2024 Red Hat, Inc.
```

File: packages/renderer/src/lib/pod/PodEmptyScreen.spec.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import '@testing-library/jest-dom/vitest';
+
+import { afterEach } from 'node:test';
```
Comment (feloy) [inline packages/renderer/src/lib/pod/PodEmptyScreen.spec.ts]:
```suggestion
import { afterEach } from 'vitest';
```

**Review by feloy — CHANGES_REQUESTED** [review]

**Review by feloy — APPROVED** [review]
LGTM

I don't find very demonstrative to start a pod with only the infra container. I think it would be better to start a pod with some payload

**Comment by dgolovin** [comment]
> LGTM
> 
> I don't find very demonstrative to start a pod with only the infra container. I think it would be better to start a pod with some payload

Sounds good to me, because without payload even terminal cannot function. Will look into it.

**Comment by benoitf** [comment]
setting draft mode as there will be changes

**Review by feloy — APPROVED** [review]
LGTM
Thanks for adding the hello container

**Pull request merged**